pkgs/applications/networking/cluster/terraform-providers/cloudfoundry/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "terraform-provider-cloudfoundry";
+  version = "0.12.4";
+
+  src = fetchFromGitHub {
+    owner = "cloudfoundry-community";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1mgsz0jj46wwq6a6rvx5v942kdckxj43r9kjgps1rh55maghsr9y";
+  };
+
+  vendorSha256 = "01lfsd9aw9w3kr1a2a5b7ac6d8jaij83lhxl4y4qsnjlqk86fbxq";
+
+  # needs a running cloudfoundry
+  doCheck = false;
+
+  postInstall = "mv $out/bin/terraform-provider-cloudfoundry{,_v${version}}";
+
```

Could we possibly add the `provider-source-address` attribute.

```suggestion
passthru = { provider-source-address = "registry.terraform.io/cloudfoundry-community/cloudfoundry"; };
```

This attribute is used to create a shim for terraform 0.13 compatibility.

https://github.com/nixos/nixpkgs/blob/39e62d5d6e26668a818fdbdae46c4f571447f4e5/pkgs/applications/networking/cluster/terraform/default.nix#L72-L80